src/jetson/models/SSD/ssd.py

```diff
@@ -129,6 +126,7 @@ def load_weights(self, base_file):
 # This function is derived from torchvision VGG make_layers()
 # https://github.com/pytorch/vision/blob/master/torchvision/models/vgg.py
 def vgg(cfg, i, batch_norm=False):
+    '''This is the backbone used for feature extraction'''
```

Comment:
can you move these 2 comments into the docstring. also provide documentation on parameters in this function